### DISCUSSION TITLE
Need Your Help - Cannot Access Localhost After Install

### DISCUSSION THREAD

**Discussion by Wildcat1** [koush/scrypted#245] [Q&A]
I've tried installing this for MAC using the instructions here (https://github.com/koush/scrypted/wiki/Installation:-Mac) and even though that appears to complete correctly whenever I try and access https://localhost:10443/ it just fails, no request to allow certificate etc, nothing.Help in resolving this is appreciated. Running this on a Mac with Catalina. For reference, I have HomeBridge running without issue. Thanks

For reference, here is the last command line I see listed before the Scrypted install ends:

==> Installing gst-plugins-ugly
==> Pouring gst-plugins-ugly--1.20.1.catalina.bottle.tar.gz
🍺 /usr/local/Cellar/gst-plugins-ugly/1.20.1: 58 files, 1.7MB
==> Running brew cleanup gst-plugins-ugly...
Error during previous command. Ignoring.

Also see this:

echo "Installing Scrypted Launch Agent..."

RUN mkdir -p ~/Library/LaunchAgents

NODE_16_PATH=$(brew --prefix node@16)
if [ ! -d "$NODE_16_PATH" ]
then
    echo "Unable to determine node@16 path."
    exit 1
fi

NODE_16_BIN_PATH=$NODE_16_PATH/bin
if [ ! -d "$NODE_16_BIN_PATH" ]
then
    echo "Unable to determine node@16 bin path."
    echo "$NODE_16_BIN_PATH does not exist."
    exit 1
fi

export PATH=$NODE_16_BIN_PATH:$PATH

NPX_PATH=$NODE_16_BIN_PATH/npx
if [ ! -f "$NPX_PATH" ]
then
    echo "Unable to find npx."
    exit 1
fi

echo "Installing Scrypted..."
RUN $NPX_PATH -y scrypted install-server

BREW_PREFIX=$(brew --prefix)
if [ -z "$BREW_PREFIX" ]
then
    echo "Unable to determine brew prefix."
    exit 1
fi

**Comment by cheme75** [comment]
Same on my win pc, seems like a common problem but not finding much info on what to try. Someone said discord, but not finding much there either.

**Comment by Wildcat1** [comment]
> I've tried installing this for MAC using the instructions here (https://github.com/koush/scrypted/wiki/Installation:-Mac) and even though that appears to complete correctly whenever I try and access https://localhost:10443/ it just fails, no request to allow certificate etc, nothing.Help in resolving this is appreciated. Running this on a Mac with Catalina. For reference, I have HomeBridge running without issue. Thanks
> 
> For reference, here is the last command line I see listed before the Scrypted install ends:
> 
> ==> Installing gst-plugins-ugly ==> Pouring gst-plugins-ugly--1.20.1.catalina.bottle.tar.gz 🍺 /usr/local/Cellar/gst-plugins-ugly/1.20.1: 58 files, 1.7MB ==> Running brew cleanup gst-plugins-ugly... Error during previous command. Ignoring.
> 
> Also see this:
> 
> echo "Installing Scrypted Launch Agent..."
> 
> RUN mkdir -p ~/Library/LaunchAgents
> 
> NODE_16_PATH=$(brew --prefix node@16) if [ ! -d "$NODE_16_PATH" ] then echo "Unable to determine node@16 path." exit 1 fi
> 
> NODE_16_BIN_PATH=$NODE_16_PATH/bin if [ ! -d "$NODE_16_BIN_PATH" ] then echo "Unable to determine node@16 bin path." echo "$NODE_16_BIN_PATH does not exist." exit 1 fi
> 
> export PATH=$NODE_16_BIN_PATH:$PATH
> 
> NPX_PATH=$NODE_16_BIN_PATH/npx if [ ! -f "$NPX_PATH" ] then echo "Unable to find npx." exit 1 fi
> 
> echo "Installing Scrypted..." RUN $NPX_PATH -y scrypted install-server
> 
> BREW_PREFIX=$(brew --prefix) if [ -z "$BREW_PREFIX" ] then echo "Unable to determine brew prefix." exit 1 fi

@koush any ideas ? Thanks

**Reply by Wildcat1** [reply]
Tried cleaning up perms and running a reinstall of HomeBrew then also re-ran the Scrypted install but now see this:

Running: brew update
Already up-to-date.
Running: brew install node@16
Warning: node@16 16.15.0 is already installed and up-to-date.
To reinstall 16.15.0, run:
  brew reinstall node@16
Running: brew install sdl2
Warning: sdl2 2.0.22 is already installed and up-to-date.
To reinstall 2.0.22, run:
  brew reinstall sdl2
Running: brew install gstreamer gst-plugins-base gst-plugins-good gst-plugins-bad gst-plugins-ugly
Warning: gstreamer 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gstreamer
Warning: gst-plugins-base 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gst-plugins-base
Warning: gst-plugins-good 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gst-plugins-good
Warning: gst-plugins-bad 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gst-plugins-bad
Warning: gst-plugins-ugly 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gst-plugins-ugly
Running: brew install gst-python
Warning: gst-python 1.20.1 is already installed and up-to-date.
To reinstall 1.20.1, run:
  brew reinstall gst-python
Running: pip3 install --upgrade pip
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/bin/pip3", line 7, in <module>
    from pip._internal.cli.main import main
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/cli/main.py", line 9, in <module>
    from pip._internal.cli.autocompletion import autocomplete
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/cli/autocompletion.py", line 10, in <module>
    from pip._internal.cli.main_parser import create_main_parser
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/cli/main_parser.py", line 8, in <module>
    from pip._internal.cli import cmdoptions
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/cli/cmdoptions.py", line 24, in <module>
    from pip._internal.cli.parser import ConfigOptionParser
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/cli/parser.py", line 12, in <module>
    from pip._internal.configuration import Configuration, ConfigurationError
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/configuration.py", line 20, in <module>
    from pip._internal.exceptions import (
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_internal/exceptions.py", line 14, in <module>
    from pip._vendor.rich.console import Console, ConsoleOptions, RenderResult
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pip/_vendor/rich/console.py", line 8, in <module>
    from dataclasses import dataclass, field
ModuleNotFoundError: No module named 'dataclasses'

**Comment by michael-murdza** [comment]
@Wildcat1  -  did you ever figure this out?
